Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1332,6 +1332,11 @@ void HTMLElement::checkAndPossiblyClosePopoverStackInternal()
 // https://html.spec.whatwg.org/#popover-focusing-steps
 static void runPopoverFocusingSteps(HTMLElement& popover)
 {
+    if (is<HTMLDialogElement>(popover)) {
+        downcast<HTMLDialogElement>(popover).runFocusingSteps(true);
```

Comment:
I commented on the spec PR, but I think running this with `isModal` set to `true` is wrong here.

Reply:
I adjusted the patch while waiting for the spec PR issue discussion to be concluded. Do note I adjusted popover-dialog-initial-focus.html to what I think is expected after your suggestion above, hopefully the spec discussion will conclude the same and update the test like that.